### DISCUSSION TITLE
カテゴリーについて

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#55] [要望・アイデア・不具合]
よくよく思ったんですが、カテゴリーの
「要望・アイデア・不具合」
は、

1. 「要望・アイデア・改善」
2. 「不具合」
3. 「障害（ユーザーからの障害報告）」

て分けたほうがいいような気がしましたが、どうでしょう？？😣

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！

まず一旦、私たちがこのカテゴリー分けにした経緯をお伝えします

- `ユーザーさまから運営に対して要望を伝えるカテゴリー`は間違いなく作ろうと思っていて、ラベル付けや詳細さについて議論をしていた
- 話に出ていたパターンはおよそ以下のようなもの
    - 今運用しているように、全部をひっくるめた1カテゴリー
        - `要望・アイデア・不具合`
    - blonoさまから提案いただいているような、`より良くする意見`と`マイナスをゼロに戻す`カテゴリー
        - `要望・アイデア`
        - `不具合・バグ`
- 別の観点として`運営が公式に発信する障害情報`を作りたかった
    - このカテゴリーにおいては不確かな情報は載せないようにして「ここを見れば大丈夫」な状態を作りたいという思い

これらを複合的に考えている中で

- ユーザーさまからの障害情報（不具合も含む）と公式からの障害情報が別カテゴリーとして存在するとややこしそう
- 一見するとバグっぽいのだけれど、過去の色々な判断もあって「仕様です」と答えざるを得ないような箇所もある
    - その場合、カテゴリーを`不具合`から`要望`に変えていただくのも面倒なはず
    - 投稿するときに「これは本当に不具合なのか、（使いづらいにせよ）正しい仕様なのか」を判断していただくのも大変

といった話になり、現在の`要望・アイデア・不具合`とひとまとめにする選択をしました

---

クローズドベータテストの間は引き続きこちらのカテゴリー分けで進めさせていただければと考えています
ただ、もう少し人数を増やしたテストの実施を現在検討中でして、そこで`不具合・バグ`系のDiscussionの在り方をお話できればと思います！
